.github/workflows/duplicates.yml

```diff
@@ -0,0 +1,20 @@
+name: Check Patch Duplicates
+on:
+  pull_request:
+    branches: [ master, Development ]
+
+# Restrict the permissions of the ephemeral GitHub token used by this workflow
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Check for duplicate virtual file paths
+      run: |
+        python3 DupeFinder.py -c
```

Comment:
Where is `DupeFinder.py` itself?

Reply:
It is in the root of the repository, added by 3330.